hello-world/AKS/Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM ubuntu:22.04
-RUN apt-get update && apt-get -y upgrade && apt-get -y install nginx python3.10
+FROM mcr.microsoft.com/devcontainers/base:dev-ubuntu-22.04 AS build
+ENV PROJECT_DIR=~/
+WORKDIR ${PROJECT_DIR}/confidential-container-demos
+COPY . ./
+RUN chmod +x hello-world/get_verbose_report.sh && ./hello-world/get_verbose_report.sh
+
+FROM mcr.microsoft.com/cbl-mariner/base/python:3.9
+RUN tdnf update -y && tdnf -y upgrade && tdnf -y install nginx
 
 WORKDIR /app
 
 # copy all files from this folder to working directory (ignores files in .dockerignore)
-COPY . /app/
-RUN mv default /etc/nginx/sites-available/default
+ENV BUILD_DIR=~/confidential-container-demos/hello-world/AKS
```

Comment:
Should this be an ARG instead of ENV?  ENV will make this available to the process at runtime.